azure/src/services/azureFunctionCloudService.ts

```diff
@@ -53,20 +54,10 @@ export class AzureFunctionCloudService implements CloudService {
   public containerResolver: ContainerResolver;
 
   /**
-   *
-   * @param name Name of function to invoke
-   * @param fireAndForget Wait for response if false (default behavior)
-   * @param payload Body of HTTP request
-   * @param headers Headers of the context
-   * @param params StringParams with values for URL
+   * @param params invoke interface with parameters needed
    */
-  public async invoke<T>(
-    name: string,
-    fireAndForget,
-    payload: any = null,
-    headers: StringParams = new StringParams(),
-    params: StringParams = new StringParams()
-  ) {
+  public async invoke<T>( params: InvokeRequest) {
+    const {name, fireAndForget, payload, headers, pathParams} = params
```

Comment:
nit: add spaces between the curly braces `{ name` and `pathParams }`

Reply:
Also add missing semicolon

Reply:
Added in my next commit thanks for this feedback